orchestrator/src/main/resources/db/migration/orchestrator/V2004__create_system_setting_table.sql

```diff
@@ -0,0 +1,25 @@
+-- ====================================================================
+-- Purpose: Create the system_setting table
+-- Database: PostgreSQL
+-- ====================================================================
+-- Enable pgcrypto extension for gen_random_uuid()
+CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+
+CREATE TABLE IF NOT EXISTS system_setting (
```

Comment:
We should enforce the single-row constraint

Use a fixed UUID for the singleton row